emr_launcher.py

```diff
@@ -97,16 +116,19 @@ def launch_cluster(event: dict = {}, context: object = None) -> dict:
 
     cluster_config = read_config("cluster")
 
+    cluster_config.update(read_config("configurations"))
```

Comment:
`configurations` isn't required for a cluster to come up, so this should have `False` passed in, as the `steps` config does below.

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/f0e5b01f19ec0daebd2ef9849b82c5dc09440394